lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -104,13 +118,24 @@ defmodule MembraneLiveWeb.EventChannel do
     end
   end
 
-  defp join_event_stream(socket) do
+  defp get_event_pid(socket) do
```

Comment:
I would rename this function as it doesn't return event pid.

Reply:
Fixed.